debian/kafka-connect/Dockerfile

```diff
@@ -39,3 +39,6 @@ RUN echo "===> Installing JDBC, Elasticsearch and Hadoop connectors ..." \
 
 RUN echo "===> Installing GCS Sink Connector ..."
 RUN confluent-hub install confluentinc/kafka-connect-gcs:latest --no-prompt
+
+ENV CONNECT_PLUGIN_PATH=/usr/share/java/,/usr/share/confluent-hub-components/
```

Comment:
This is good: the `/usr/share/java/` is correct, and the `/usr/share/confluent-hub-components` is where the Confluent Hub client will install additional connectors.

Reply:
Actually, should this be set in `debian/kafka-connect-base` instead of here? That way images based on that image (`cp-kafka-connect` and customer-specific images) will have this property set correctly.

Reply:
Done